scripts/api/entity/planet.lua

```diff
@@ -131,8 +131,11 @@ function Entity:setAxialRotationTime(rotation_time)
     end
     return self
 end
---- Sets a SpaceObject around which this Planet orbits, as well as its orbital period in orbital degrees per tick.
---- Example: moon:setOrbit(planet, 20)
+--- Sets a SpaceObject around which this SpaceObject orbits, as well as its orbital period in seconds. Setting time to 0 will stop movement, while still being locked in orbit.
+--- An orbit can be cancelled by setting the SpaceObject as its own target.
```

Comment:
I think it's better to have `obj:setOrbit(nil)` cancel the orbit. Which can be done by deleting the orbit component:
`obj.components.orbit = nil`

Reply:
Oh I thought I tried that, obviously not. Done